libutp/utp_internal.cpp

```diff
@@ -3416,7 +3416,7 @@ utp_process_udp(utp_context *ctx, const byte *buffer, size_t len,
 
       return 1;
     }
-
+    /*
```

Comment:
Probably best to remove code if it's not going to be used, rather than commenting it out.

Reply:
disagree, we may want to re-enable it later (maybe)

Reply:
I suppose I can live with that for now, but that policy merits further discussion.